web-components/src/components/inputs/new/TextAreaField/TextAreaField.ChartCounter.tsx

```diff
@@ -5,13 +5,16 @@ type Props = {
   charLimit?: number;
 };
 
-const DEFAULT_CHAR_LIMIT = 160;
+export const DEFAULT_CHAR_LIMIT = 160;
 
 export const TextAreaFieldChartCounter = ({
   value,
   charLimit = DEFAULT_CHAR_LIMIT,
 }: Props) => {
-  const charsLeft = (charLimit || Infinity) - ((value && value.length) || 0);
+  const charsLeft = Math.max(
+    0,
+    (charLimit || Infinity) - ((value && value.length) || 0),
+  );
```

Comment:
it looked weird to display negative values for the number of characters left

Reply:
Indeed, that's a good improvement.